src/EnergyPlus/HVACManager.cc

```diff
@@ -665,10 +666,11 @@ void SimHVAC(EnergyPlusData &state)
     bool MonotonicIncreaseFound;
     bool MonotonicDecreaseFound;
 
-    Array1D<Real64> const ConvergLogStackARR(DataConvergParams::ConvergLogStackDepth, {0.0, -1.0, -2.0, -3.0, -4.0, -5.0, -6.0, -7.0, -8.0, -9.0});
-    Real64 const sum_ConvergLogStackARR(sum(ConvergLogStackARR));
-    Real64 const square_sum_ConvergLogStackARR(pow_2(sum_ConvergLogStackARR));
-    Real64 const sum_square_ConvergLogStackARR(sum(pow(ConvergLogStackARR, 2)));
+    static constexpr std::array<Real64, DataConvergParams::ConvergLogStackDepth> ConvergLogStackARR = {
```

Comment:
Most of these changes are self-explanatory, but there are a few that could use a look-over by reviewers. These HVACManager changes are one of those places.